### DISCUSSION TITLE
How to catch all eMails from other systems

### DISCUSSION THREAD

**Discussion by XtraLarge** [Mailu/Mailu#3134] [User support]
Hi,
I'm using many internal linux systems under the domain "fritz.box". Often they send emails from their hostname, for example "root@proxy.fritz.box". How do I configure mailu to accept all those eMails (I think with relay network) and catch them to a system-maintenace eMail Account.

**Comment by teadur** [comment]
Add one master domain for fritz.box and all hosts as alternatives to fritz.box ex host.fritz.boc after that add catch all alias for *@fritz.box and send it to smt@mail.tld address

**Reply by XtraLarge** [reply]
Yes, but in that case I muste configure mailu everytime I create a new system. That's to prone to errors.
